src/switch.jsx

```diff
@@ -0,0 +1,29 @@
+
+import React from 'react';
+import './App.css';
+import './App.jsx'
+
+const Switch = ({ sortAsc, handleSortToggleClick }) => {
+    return (
+        <>
+
+            <input
+                className="react-switch-checkbox"
+                id={`react-switch-new`}
```

Comment:
prefer `id='react-switch-new'` here unless you intend to use string interpolation